### DISCUSSION TITLE
Can we have an else while() for a conditional block please?

### DISCUSSION THREAD

**Discussion by VladimirBangPow** [nodejs/help#5013] [Ideas]
Can we have an else while() for a conditional block please?

if(condition){
}else while(condition2){

}else{
}

Also! why not?
while(condition1){
}else{
}

And, finally:

while(condition1){

}else while(condition2){

}

**Comment by cjihrig** [comment]
Technically, you can already use a `while` loop as the `else` condition, but everything else you're asking for would require changes to the language. Node can't do that, but feel free to propose this at https://github.com/tc39. If your changes make it into the language, they would automatically make their way into Node.
